main.c

```diff
@@ -273,7 +273,7 @@ static void keyboard_handle_keymap(void *data, struct wl_keyboard *wl_keyboard,
 	case WL_KEYBOARD_KEYMAP_FORMAT_NO_KEYMAP:
 		seat->xkb_keymap = xkb_keymap_new_from_names(seat->state->xkb_context, NULL, XKB_KEYMAP_COMPILE_NO_FLAGS);
 		break;
-	case WL_KEYBOARD_KEYMAP_FORMAT_XKB_V1:;
```

Comment:
Unfortunately we need to keep this one because of a C language quirk: variable declarations can't follow a `case`.